lib/util.dart

```diff
@@ -153,6 +153,8 @@ class Util {
   /// This actually creates an [EmbeddedGameWidget] with a [SimpleGame] whose only content is an [AnimationComponent] created from the provided [animation].
   /// You can use this implementation as base to easily create your own widgets based on more complex games.
   /// This is intended to be used by non-game apps that want to add a sprite sheet animation.
+  ///
+  /// @Deprecated('Use FlameSpriteAnimation instead')
```

Comment:
This (and the other one) shouldn't be comments right?

Reply:
As far as I know, that is how you deprecate stuff in dart, but I could be wrong

Reply:
Surely it must be without the /// before it?

Reply:
Yeah, I don't know what was going on my head, fixed it.